ci/docker/linux-apt-docs.dockerfile

```diff
@@ -48,16 +61,20 @@ RUN apt-get update -y && \
         libxml2-dev \
         meson \
         ninja-build \
+        nodejs \
```

Comment:
If we are installing nodejs already why not use something like https://github.com/mermaid-js/mermaid-cli instead of chromium?